docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner works
+
+## Contrastive metric learning
+
+From an algorithmic perspective,
+**Finetuner** leverages a contrastive metric learning approach to improve your model.
+How does it work?
+
+### Step 1: Convert a model into an Embedding Model
+
+Finetuner interprets the backbone model architecture,
+removes the default *head*, applies *pooling* and freezes layers that do not need to be trained..
+For example, if your pre-trained model is trained for cat-dog classification,
```

Comment:
for an image classification task (e.g. cats and dogs), Finetuner is going to remove the classification head (cat-dog classifier) and turn ...